docs/config/appearance.md

```diff
@@ -1,4 +1,27 @@
-### Color Scheme
+- [Colors \& Appearance](#colors--appearance)
```

Comment:
I looked into breaking it up into a separate page, I think the content on the page works well being grouped on the same page together since it's all under the category of appearance/styling. 

I didn't consider being an issue because i assumed that docs wouldn't change much but I recognize it could creates extra work down the road. I'll remove it from the revised PR.

Here's a screen shot of what it looks like incase it's something you want to revisit in the future.

![colors-appearance-toc](https://user-images.githubusercontent.com/61954919/215383464-7c87499c-3004-440f-bdc8-7f5b80b6ebaf.jpeg)